core-standards.php

```diff
@@ -47,4 +47,5 @@ function classloader($class) {
 if (defined('WP_CLI') && WP_CLI) {
   \WP_CLI::add_command('user export-csv', __NAMESPACE__ . '\Commands\UserExport');
   \WP_CLI::add_command('user import-csv-raw', __NAMESPACE__ . '\Commands\UserImport');
+  \WP_CLI::add_hook('before_run_command', __NAMESPACE__ . '\Hooks\MultilingualSupport::changeLanguage');
```

Comment:
This is only affecting WP-CLI and no regular runtime functionality, so I would keep it in `Commands`.

I'd also rename it to Sitepress to implicitly clarify that it's only integrating with WPML and not other multilingual solutions.
```suggestion
  \WP_CLI::add_hook('before_run_command', __NAMESPACE__ . '\Commands\Sitepress::changeLanguage');
```

Alternatively, if you intend to support others in the future:
```suggestion
  \WP_CLI::add_hook('before_run_command', __NAMESPACE__ . '\Commands\Language::limitLanguage');
```

Reply:
Personally I'd rather stick with the structure as is since there might be some more hooks to come in the future and also to keep it consistent. If moving it under the command namespace it won't be very clear since the namespace imply to expect something else.

I chose a more generic name in case we want to extend the support for e.g. Polylang we wouldn't have to touch this line again and only adjust the method itself. But I'll rename the class and method to your suggestion :-)